pkg/devcontainer/sshtunnel/sshtunnel.go

```diff
@@ -103,7 +103,7 @@ func ExecuteCommand(ctx context.Context, client client2.WorkspaceClient, agentIn
 			}
 		}
 
-		writer := log.Writer(logrus.InfoLevel, false)
+		writer := log.Writer(logrus.ErrorLevel, false)
```

Comment:
We shouldn't change this to `ErrorLevel` here as all log output will be piped over stderr from the agent command because stdout is used for the connection. 
I checked it locally and the post create command output still shows up as `error` when this is info. 
So we could either revert it or implement a proper check on this writer and see which log level the agent command sent originally

Reply:
yes but this writer is specifically used for the STDERR in the devssh.Run function
And usually if something is in stderr, it should be an error

Probably just a nitpick on my side, I can revert if this creates problems let me know